cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1690,7 +1690,16 @@ public MNode getMNode(MNode deviceMNode, String measurementName) {
     ExecutorService pool =
         new ThreadPoolExecutor(
             THREAD_POOL_SIZE, THREAD_POOL_SIZE, 0, TimeUnit.SECONDS, new LinkedBlockingDeque<>());
-    List<PartitionGroup> globalGroups = metaGroupMember.getPartitionTable().getGlobalGroups();
+
+    List<PartitionGroup> globalGroups = new ArrayList<>();
+    if (IoTDBConstant.PATH_ROOT.equals(plan.getPath().getFullPath())
+        || plan.getPath().getFullPath().contains(IoTDBConstant.PATH_WILDCARD)) {
```

Comment:
As we have supported special characters, how about  the timeseries contains asterisk like the following:
`root.sg."device_1*".sensor1`

Reply:
This grammar....  I alter this, support this grammar...